app/views/static/_home_cover.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="cover">
+  <div class="cover-image" style="background-image: url(<%= asset_path 'static/home-cover-04.jpg' %>)"></div>
```

Comment:
I was experimenting with a few possible ideas to animate the cover background and text. I can move the background-image to css for now.